torch/testing/_internal/common_optimizers.py

```diff
@@ -987,6 +1036,15 @@ def optim_error_inputs_func_sparseadam(device, dtype):
                 "TestOptimRenewed",
                 "test_set_default_dtype_works_with_foreach",
             ),
+            DecorateInfo(
+                toleranceOverride(
+                    {  # previously atol=1-05, rtol=1.3e-06,
```

Comment:
I don't think this comment helps much beyond in the context of this PR.
Feel free to either add context about what previously means or remove it :)